scripts/hid_configurator/configurator_core.py

```diff
@@ -219,30 +225,41 @@ def create_fetch_report(recipient, event_id):
         a configuration value from a device.
         Recipient is a device product ID. """
 
-    assert(type(recipient) == int)
-    assert(type(event_id) == int)
+    assert type(recipient) == int
+    assert type(event_id) == int
 
     status = ConfigStatus.FETCH
     report = struct.pack('<BHBBB', REPORT_ID, recipient, event_id, status, 0)
 
-    assert(len(report) <= REPORT_SIZE)
+    assert len(report) <= REPORT_SIZE
     report += b'\0' * (REPORT_SIZE - len(report))
 
     return report
 
 
+def check_range(value, value_range):
+    return value_range[0] <= value <= value_range[1]
```

Comment:
this seems incorrect

Reply:
unlike C, all comparison operations in Python have the same priority
https://docs.python.org/2/reference/expressions.html#comparisons

Reply:
you are right, thanks for pointing this out